server/.gitignore

```diff
@@ -39,3 +39,6 @@ app_data/
 
 # report builder server credentials
 report_builder/config.yaml
+
+# fridge tag imports
+sensor_*.txt
```

Comment:
Thanks !

Reply:
I wondered too if we should clean up these file at some stage 🤔